client/modules/datafiles/src/DatafilesModal/PreviewModal/PreviewModal.tsx

```diff
@@ -70,55 +70,61 @@ export const PreviewModalBody: React.FC<{
           justifyContent: 'center',
         }}
       >
-        {scheme === 'private' && api === 'tapis' && (
-          <MoveModal
+        {scheme === 'private' &&
+          api === 'tapis' &&
+          !selectedFile.path.endsWith('.hazmapper') && (
```

Comment:
Within the preview modal, there is another file menu toolbar handled by this logic here. Is there a high level component where I could add logic so that all file menu options are either on or off wherever that menu could appear?

Reply:
Ah gotcha. Since here you have `!selectedFile.path.endsWith('.hazmapper')` on every toolbar button, we could instead wrap all toolbar buttons in `<></>`, and conditionally render that with `!selectedFile.path.endsWith('.hazmapper')`

Reply:
Yes definitely! Added that change. Thanks!